Compiler warnings control reaches end of non-void function

Compiler warning:
```
src/Pattern.h:30:40: warning: control reaches end of non-void function [-Wreturn-type]
        int height(int h) { size(width(), h); }
                                              ^
```

--- a/src/Pattern.h
+++ b/src/Pattern.h
@@ -27,7 +27,7 @@ public:
 	void width(int w) { size(w, height()); }
 
 	int height() const { return mPatternGrid.size(); }
-	int height(int h) { size(width(), h); }
+	void height(int h) { size(width(), h); }
 
 	/**
 	 * Changes the size of the pattern.

